### ISSUE TITLE
MapList NG2 - mobile (iPhone) friendly - coool

### ISSUE THREAD

**Issue by sinisarudan** [mprinc/KnAllEdge#110]
- ng1 -> ng2
- actions buttons
- regular and moderator view
- toolbar
- adjustable to a small screen

**Comment by sinisarudan** [comment]
@mprinc : what to do on a small screen when all columns cannot fit in a row? to scroll left right or put them in several rows?

**Comment by sinisarudan** [comment]
@mprinc will we add sorting by clicking on column headers (name, Creation date, Type, Id)?
or use some component that support it natively?

**Comment by mprinc** [comment]
it is better one under another then scrolling left/right

**Comment by mprinc** [comment]
i am not aware of any, but i guess the best is to make someone supporting that eventually, but no rush now

**Comment by mprinc** [comment]
Then the best seems to me that we can click on the column name and then we get an icon up/down
like here: http://fontawesome.io/icon/sort-desc/ or https://design.google.com/icons/#ic_keyboard_arrow_down

**Comment by sinisarudan** [comment]
I agree, I thought the same. I am creating design for wide-screen and mobile version

**Comment by sinisarudan** [comment]
DONE:
ng1 -> ng2
small screen visibility and all UI interactions and functionality

**Comment by sinisarudan** [comment]
I used **flexbox-based** approach for **adding headers in front of each item property in mobile-view*.
This creates more complex data-structure, so we can think also of **div:before addition** approach, with which we should solve equal header-column width:
-----------
/* another approach for headers in mobile-version
			&:before {
				width: 100px; //35%;
				padding-right: 10px;
				white-space: nowrap;
				border-right: 1px solid darken($item_background_color, 35%);
			}
			*/

+ ...

/* another approach for headers in mobile-version
			.item_property:nth-of-type(2):before { content: "Id"; }
			.item_property:nth-of-type(3):before { content: "Actions"; }
			.item_property:nth-of-type(4):before { content: "Authors"; }
			.item_property:nth-of-type(5):before { content: "Type"; }
			.item_property:nth-of-type(7):before { content: "Created"; }
			*/

**Comment by mprinc** [comment]
For the moment do not change if it works as it is now

**Comment by sinisarudan** [comment]
@mprinc : not change what?

**Comment by sinisarudan** [comment]
fixed missing `clone` awesome icon for mapDuplicate action-button; used another one, nicer, icon

**Comment by sinisarudan** [comment]
DONE earlier: showing participants list in a dialog
